CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ endmacro()
 
 # Enabling all warnings in MSVC spams too much
 if(NOT MSVC)
-	add_definitions(-Wall)
+	add_definitions(-Wall -Wsign-conversion)
```

Comment:
I mainly added it because I couldn't find an equivalent warning in visual studio and wanted to see how many things it caught.